### PR TITLE
Revised monitoring doc

### PR DESCRIPTION
**Pull request by rmatharu**

### CODE REVIEW

**Comment by vjagadish1989** [comment]
Can u attach a preview like @cameronlee314 did in his PR?

**Comment by vjagadish1989** [comment]
Also, can you also fix the link on the menu for monitoring?

**Comment by rmatharu** [comment]
screen shot 2018-09-26 at 11 08 42 am
screen shot 2018-09-26 at 11 08 37 am
screen shot 2018-09-26 at 11 08 34 am
screen shot 2018-09-26 at 11 08 27 am
screen shot 2018-09-26 at 11 08 24 am
screen shot 2018-09-26 at 11 08 20 am
screen shot 2018-09-26 at 11 08 13 am
screen shot 2018-09-26 at 11 08 08 am
screen shot 2018-09-26 at 11 08 04 am
screen shot 2018-09-26 at 11 07 57 am
screen shot 2018-09-26 at 11 07 55 am
screen shot 2018-09-26 at 11 07 46 am

**Comment by rmatharu** [comment]
> Can u attach a preview like @cameronlee314 did in his PR?

Done

> Also, can you also fix the link on the menu for monitoring?
Done

File: docs/learn/documentation/versioned/index.html [hunk 1]
```diff
@@ -37,7 +37,54 @@ <h4>API</h4>
...
+
+<h4>API</h4>
+
+<ul class="documentation-list">
+  <li><a href="api/overview.html">Overview</a></li>
+</ul>
+
+<h4>Core</h4>
+
+<ul class="documentation-list">
```
Comment (vjagadish1989) [inline docs/learn/documentation/versioned/index.html]: Is this a result of an incorrect merge? for eg: These links should not be on the new website structure

Comment (rmatharu) [reply docs/learn/documentation/versioned/index.html]: done

**Pull request closed without merging**